### PR TITLE
update to lotus 1.13

### PR DESCRIPTION
**Pull request by frrist**
gets us ready for lotus 1.13 when it lands
TODO:
- [x] update to specs-actors v6 when cut
- [x] update to 1.13 (instead of RC) lotus release when cut
- [x] validate lily can sync with the calibration network

### CODE REVIEW

File: chain/actors/builtin/miner/diff.go [hunk 1]
```diff
@@ -4,7 +4,8 @@ import (
 	"context"
 
 	cbg "github.com/whyrusleeping/cbor-gen"
-	"go.opentelemetry.io/otel/api/global"
```
Comment (frrist) [inline chain/actors/builtin/miner/diff.go:7]: the bulk of changes in PR are these: upgrading from v0.12 to v0.20 of opentelemetry

Comment (iand) [reply chain/actors/builtin/miner/diff.go:7]: Wonder if we could do these changes first in a separate PR?

Comment (frrist) [reply chain/actors/builtin/miner/diff.go:7]: Good idea: https://github.com/filecoin-project/lily/pull/749

File: commands/daemon.go [hunk 2]
```diff
@@ -238,7 +238,7 @@ Note that jobs are not persisted between restarts of the daemon. See
 
 			node.Override(new(dtypes.Bootstrapper), isBootstrapper),
 			node.Override(new(dtypes.ShutdownChan), shutdown),
-			node.Online(),
+			node.Base(),
```
Comment (placer14) [inline commands/daemon.go:241]: What is the functional difference between the old and new injection?

File: commands/lily.go [hunk 3]
```diff
@@ -43,6 +43,7 @@ func LilyNodeAPIOption(out *lily.LilyAPI, fopts ...node.Option) node.Option {
 	resAPI := &lily.LilyNodeAPI{}
 	opts := node.Options(
 		node.WithRepoType(repo.FullNode),
+		node.WithEnableLibp2pNode(true),
```
Comment (placer14) [inline commands/lily.go:46]: Any reason this is now required?

File: commands/daemon.go (hunk 2 above)
Comment (frrist) [reply commands/daemon.go:241]: Before: https://github.com/filecoin-project/lotus/blob/v1.11.0/node/builder.go#L457-L473
After: https://github.com/filecoin-project/lotus/blob/v1.13.0/node/builder.go#L223-L235
A name change and a different way of enabling the libp2p bits.

**Pull request merged**